src/NexusMods.App.UI/Controls/Spine/SpineViewModel.cs

```diff
@@ -71,11 +79,26 @@ public class SpineViewModel : AViewModel<ISpineViewModel>, ISpineViewModel
                 .DisposeWith(disposables);
 
             Home.Click = ReactiveCommand.Create(NavigateToHome);
-
             Downloads.Click = ReactiveCommand.Create(NavigateToDownloads);
 
-            // For now just select home on startup
-            NavigateToHome();
+            workspaceController
+                .WhenAnyValue(controller => controller.ActiveWorkspace)
+                .Select(workspace => workspace?.Context)
+                .Select(context =>
+                {
+                    if (context is LoadoutContext loadoutContext)
+                    {
+                        return new LoadoutLeftMenuViewModel(
+                            loadoutContext,
+                            workspaceController,
+                            serviceProvider
+                        );
```

Comment:
This will have the Apply button in the future, so I don't think that creating and discarding it each time the Loadout changes is a good idea. 

Would you consider having the left menu VM be part of the Workspace instead?
Something like a Property that contains the leftmenu that gets populated when the workspace is created and is kept alive in the workspace?

Keep in mind that these left menus will get busier as we add things: 
![image](https://github.com/Nexus-Mods/NexusMods.App/assets/26797547/59ebc91c-81d7-454a-a02c-8d6fe294f13e)

Reply:
> This will have the Apply button in the future, so I don't think that creating and discarding it each time the Loadout changes is a good idea.

Right now, we only have a launch button.

> Would you consider having the left menu VM be part of the Workspace instead?

No, we have workspaces that don't have left menus. The left menu is a main-window-concept.

Reply:
You are right, popped out workspaces don't have navigation. Doesn't mean they can't have the info anyways. Create the nav lazily if requested. 

If you still don't like that idea, I still think we want to keep the VMs alive. 

I would propose to keep the DL and MyGames directly in the Spine, while for the loadouts we could either have a map, one for each loadout, or a single one where the context gets replaced when switching. I think it would be better to keep a map for each loadout instead though.

Reply:
We can just use the factory pattern again to let some factory create a left menu depending on the current workspace context. In the `MainWindowViewModel` we'd just react to context changes in the workspace and find a VM using a factory that fits that context.

But this isn't something we should do in this PR.

Reply:
Ok, the factory idea works, as long as the spine or something then keeps the VMs alive after when the workspace is switched.